content/00.zksync-era/10.guides/10.quick-start/3.deploy-your-first-contract.md

```diff
@@ -70,37 +35,29 @@ The Solidity smart contract contains two functions:
 - `getTotalMessages` returns the number of messages stored in the smart contract.
 - `getLastMessage` returns the last message sent.
 
-::callout{icon="i-heroicons-light-bulb"}
-ZKsync Era is [EVM compatible](/zksync-protocol/glossary#evm-compatible).
-You can write smart contracts with Solidity or Vyper and use existing popular libraries like OpenZeppelin, just like on Ethereum.
-::
-
 ## Compile and deploy the contract
 
-To compile and deploy the contract you can use either Atlas or Remix:
+To compile and deploy the contract you can use either EVM or EraVM:
+
+:display_partial{path="/_partials/_eravm_vs_evm"}
+
+<br>
 
 ::content-switcher
 ---
 items: [{
-  label: 'Atlas',
-  partial: '_deploy_first/_atlas_deploy_contract'
+  label: 'EVM',
+  partial: '_deploy_first/_evm_remix_deploy_contract'
 }, {
-  label: 'Remix',
-  partial: '_deploy_first/_remix_deploy_contract'
+  label: 'EraVM',
+  partial: '_deploy_first/_eravm_remix_deploy_contract'
 }]
 ---
 ::
 
-## Check the contract in explorer
-
-Copy the smart contract address from Atlas/Remix and search it via the [%%zk_testnet_name%%
-explorer](%%zk_testnet_block_explorer_url%%). You’ll see the contract has a transaction from the message you just sent.
-
-![Contract in ZKsync explorer](/images/101-quickstart/101-contract-deployed.png)
-
 The status will be “Processed” on ZKsync Era and “Sending” on Ethereum. [Learn more about the transaction lifecycle on ZKsync](/zksync-protocol/rollup/transaction-lifecycle).
 
-In the “Contract” tab you’ll see the contract source code as Atlas and Remix automatically verified the contract for us.
+In the “Contract” tab you’ll see the contract source code as Remix automatically verified the contract for us.
```

Comment:
actually, the EVM version does not get verified. Only the EraVM as we have a Deploy & Verify option so maybe remove that part about verification

Reply:
does the automatic verification only work for EraVM?